packages/ios-client/Sources/FishjamClient/FishjamClientInternal.swift

```diff
@@ -168,13 +170,49 @@ class FishjamClientInternal {
         return audioTrack
     }
 
+    public func create(customSource: CustomSource) async throws {
```

Comment:
WDYT about less generic name? This class already has some `createxxx` methods

Reply:
This is a very common convention in Swift, the name of this function is createCustomSource, but because of the convention, it's split into the first argument.